app/services/AppealService.scala

```diff
@@ -108,16 +109,29 @@ class AppealService @Inject()(penaltiesConnector: PenaltiesConnector,
     val appealType = userRequest.answers.getAnswer[PenaltyTypeEnum.Value](SessionKeys.appealType)
     val isLPP = appealType.contains(PenaltyTypeEnum.Late_Payment) || appealType.contains(PenaltyTypeEnum.Additional)
     val agentReferenceNo = userRequest.arn
-    val penaltyNumber = userRequest.answers.getAnswer[String](SessionKeys.penaltyNumber).get
     val correlationId: String = idGenerator.generateUUID
     val userHasUploadedFiles = userRequest.answers.getAnswer[String](SessionKeys.isUploadEvidence).contains("yes")
+    if (userRequest.answers.getAnswer[String](SessionKeys.doYouWantToAppealBothPenalties).contains("yes")) {
+      multipleAppeal(enrolmentKey, appealType, isLPP, correlationId, agentReferenceNo, userHasUploadedFiles, reasonableExcuse)
```

Comment:
'multipleAppeal' would be called when we have LPP and Additional, I don't think we need this when there's LSP

Reply:
Tightened up logic to discount LSPs